src/Features/Core/Portable/Diagnostics/Analyzers/UnboundIdentifiersDiagnosticAnalyzerBase.cs

```diff
@@ -37,13 +37,28 @@ public override void Initialize(AnalysisContext context)
 
         protected static DiagnosticDescriptor GetDiagnosticDescriptor(string id, LocalizableString messageFormat)
         {
+            var customTags = DiagnosticCustomTags.Microsoft.Append(WellKnownDiagnosticTags.NotConfigurable);
```

Comment:
@Youssef1313 EnforceOnBuild only makes sense for analyzers that are in the CodeStyle layer and ship in the CodeStyle NuGet package.

Reply:
@mavasani Should I instead remove the entries from EnforceOnValues class?

https://github.com/dotnet/roslyn/blob/master/src/Analyzers/Core/Analyzers/EnforceOnBuildValues.cs#L110-L111

Reply:
That sounds reasonable.